projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -288,306 +297,311 @@
-  "item_prop.draconicevolution.walk_speed.info": "Allows you to adjust the speed boost that is applied while you are walking.",
-  "key.draconicevolution.dislocator_blink": "Advanced Dislocator Blink",
-  "key.draconicevolution.dislocator_down": "Advanced Dislocator Select Down",
-  "key.draconicevolution.dislocator_gui": "Advanced Dislocator GUI",
-  "key.draconicevolution.dislocator_teleport": "Advanced Dislocator Teleport",
-  "key.draconicevolution.dislocator_up": "Advanced Dislocator Select Up",
+  "item_prop.draconicevolution.jump_boost.info": "调整跳跃提升",
+  "item_prop.draconicevolution.jump_boost_run": "跑跳提升",
+  "item_prop.draconicevolution.jump_boost_run.info": "调整跑跳提升",
+  "item_prop.draconicevolution.mining_aoe": "范围挖掘",
+  "item_prop.draconicevolution.mining_aoe.info": "允许你调整此工具的挖掘范围。",
+  "item_prop.draconicevolution.mining_speed": "挖掘速度倍数",
+  "item_prop.draconicevolution.mining_speed.info": "允许你调整工具的挖掘速度。如果相比于速度更需要精度，这很有用。",
+  "item_prop.draconicevolution.run_speed": "疾跑速度",
+  "item_prop.draconicevolution.run_speed.info": "允许你调整疾跑速度加成。",
+  "item_prop.draconicevolution.shield_mod.always_visible": "护盾保持可见",
+  "item_prop.draconicevolution.shield_mod.always_visible.info": "纯视觉效果。如果关闭，护盾只会在吸收伤害的时候可见。",
+  "item_prop.draconicevolution.shield_mod.enabled": "开关护盾",
+  "item_prop.draconicevolution.shield_mod.enabled.info": "允许你禁用护盾。这样可以避免护盾耗电，但你也会失去保护。",
+  "item_prop.draconicevolution.walk_speed": "行走速度",
+  "item_prop.draconicevolution.walk_speed.info": "允许你调整行走时的速度。",
+  "key.draconicevolution.dislocator_blink": "高级传送符咒：闪现",
+  "key.draconicevolution.dislocator_down": "高级传送符咒：选择下一个",
+  "key.draconicevolution.dislocator_gui": "打开高级传送符咒GUI",
+  "key.draconicevolution.dislocator_teleport": "高级传送符咒：传送",
+  "key.draconicevolution.dislocator_up": "高级传送符咒：选择上一个",
   "key.draconicevolution.place_item": "放置物品",
-  "key.draconicevolution.toggle_flight": "切换飞行模式",
+  "key.draconicevolution.toggle_flight": "开关飞行",
   "key.draconicevolution.toggle_magnet": "开关物品传送符咒",
   "key.draconicevolution.tool_config": "工具配置",
   "key.draconicevolution.tool_modules": "工具模块",
-  "modular_item.draconicevolution.cant_install.level_high": "The module tier is to high for this modular item.",
-  "modular_item.draconicevolution.cant_install.not_supported": "This module is not supported by this modular item.",
-  "modular_item.draconicevolution.cant_install.wont_fit": "The module wont fit in that position.",
-  "modular_item.draconicevolution.error.module_install_limit": "Install limit reached for this module.",
-  "modular_item.draconicevolution.error.no_modular_items": "You dont have any modular items in your inventory!",
-  "modular_item.draconicevolution.requires_energy": "This item requires energy modules to function.",
-  "modular_item.draconicevolution.requires_energy_press": "Press %s to open module config.",
+  "modular_item.draconicevolution.cant_install.level_high": "模块等级过高。",
+  "modular_item.draconicevolution.cant_install.not_supported": "此模块不支持此物品。",
```

```suggestion
  "modular_item.draconicevolution.cant_install.not_supported": "此物品不支持此模块。",
```
主要是应该是“物品不支持安装此模块”而不是“模块不支持被安装在此物品上”，毕竟物品是主体，模块是附加部件